var/spack/repos/builtin/packages/python/package.py

```diff
@@ -107,6 +107,13 @@ class Python(Package):
     version("3.7.1", sha256="36c1b81ac29d0f8341f727ef40864d99d8206897be96be73dc34d4739c9c9f06")
     version("3.7.0", sha256="85bb9feb6863e04fb1700b018d9d42d1caac178559ffa453d7e6a436e259fd0d")
 
+    # Python 3.6.15 has been added back only to allow bootstrapping Spack on Python 3.6
```

Comment:
Is this necessary? I would prefer to support external-only Python 3.6.

Reply:
Is it just audits?  Could we add `#noaudit` for lines in the packages that trigger this?

Reply:
Yes, I was thinking about that. That means changes in the audit command too. I think the question is if we should do it here or we can do it in a separate PR.